frontend/src/app/(main)/articles/[id]/page.tsx

```
@@ -0,0 +1,45 @@
+      <header className="mb-10 w-full break-words text-center">
+        <h1 className="mb-3 text-4xl font-bold">{data.title}</h1>
+      </header>
+      <p className="text-right text-sm text-gray-500">
+        {formatDistanceToNow(new Date(data.created_at), { addSuffix: true, locale: ja })}に公開
+      </p>
+
+      <article className="rounded-lg bg-white p-6 shadow-md">
+        <div
+          className="prose prose-gray max-w-none break-words text-gray-800"
+          dangerouslySetInnerHTML={{ __html: data.content }}
+        />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)